src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java

```diff
@@ -296,13 +304,26 @@ void processPresence(Presence presence)
             statsId = statsIdPacketExt.getStatsId();
         }
 
+        boolean wasAudioMuted = isAudioMuted;
+        StandardExtensionElement audioMutedExt = presence.getExtension("audiomuted", "jabber:client");
```

Comment:
Ingo has been doing some work around extensions in #832 and he fixed `VideoMutedExtension`. Can you please use the same approach for audio?

Reply:
done